pkg/agent/secondarynetwork/podwatch/controller.go

```diff
@@ -77,6 +77,10 @@ type IPAMAllocator interface {
 	SecondaryNetworkRelease(podOwner *crdv1b1.PodOwner) error
 }
 
+type PodController interface {
```

Comment:
I think it should just make `podController` struct public

Reply:
updated.